pkg/services/rpcsrv/server.go

```diff
@@ -2936,3 +2938,47 @@ func (s *Server) Addresses() []string {
 	}
 	return res
 }
+
+func (s *Server) getRawNotaryPool(reqParams params.Params) (any, *neorpc.Error) {
+	if !s.chain.P2PSigExtensionsEnabled() {
+		return nil, neorpc.NewInternalServerError("P2PSignatureExtensions are disabled")
+	}
+	verbose, _ := reqParams.Value(0).GetBoolean()
+
+	nrp := s.coreServer.GetNotaryPool()
+	pool := nrp.GetVerifiedPoolTx()
+	if !verbose {
+		return pool, nil
+	}
```

Comment:
I'm encountering difficulties when attempting to test it with verbose=false. Would it be acceptable to return a map, or do I need to use a specific struct in this case?

Reply:
Ok, fixed